docs/0-einfuehrung/index.mdx

```diff
@@ -4,11 +4,14 @@ sidebar_position: 1
 ---
 
 import { StepsOverview } from '@site/src/components/StepsOverview';
+import { TeamMembers } from '@site/src/components/TeamMembers';
 
 # Einführung
 
 <StepsOverview />
 
+<TeamMembers />
```

Comment:
Wouldn't it be more convenient to have all the contents on the page? I imagine having multiple instances of an `<Person/>` component with different props such as `picUrl`, `name` and `bio`. This would avoid having to switch between files as a content editor.

Reply:
... to continue my comment from the other thread:

If we decide to ever display the authors with `<PersonCard />`'s on some other page, having the content in this particular file would mean having to duplicate content. By leaving it in `<TeamMembers />` we could drop that component in in other places.

Also, passing props in MDX wouldn't give us the support of TypeScript (as far as I know?) which is why I preferred to set the content in `<TeamMembers />`

Reply:
So, as discussed off-GitHub, we are actually going to change the approach so that content is defined in the MDX files directly. This makes sense because we want content editors to have a straightforward way to make changes. In the case of the authors it makes sense to have it here. This will also make i18n easier because we can duplicate MDX pages for each language and change the content of the props accordingly.

We will still keep some information in the Docusaurus config: the very basic metadata that is not going to change. Actually we will explore [i18n](https://docusaurus.io/docs/i18n/introduction) because there will be an English version coming and we want the metadata to multilanguage, too.

I'm going to change everything related to the authors in the next commits. For refactoring the other parts, there will be a different PR.

Anything that I forgot to summarize @vogelino?

Reply:
Great summary! Thx @dnsos ! Nothing to add! ❤️